test/visualise.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import {renderToString} from "react-dom/server";
+
+const html = component => `<html>
+<head>
+  <title>Component Preview</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+<body style="margin:0;">
+  <div id="app">
+    ${component}
+  </div>
+</body>
+</html>`;
+
+module.exports = (component, path) => afterAll(() => {
```

Comment:
This should probably be updated to ES6 module syntax:
```js
export const visualise = ...
```
Then at the end:
```js
export default visualise;
```
Or something similar anyway.

Reply:
Fixed